dotCMS/src/integration-test/java/com/dotmarketing/util/ConfigTest.java

```diff
@@ -0,0 +1,254 @@
+        System.getenv().put(DOT_TESTING_LONG, String.valueOf(Long.MAX_VALUE));
+        System.getenv().put(DOT_TESTING_BOOLEAN, String.valueOf(Boolean.TRUE));
+        System.getenv().put(DOT_TESTING_FLOAT, String.valueOf(Float.MAX_VALUE));
+        System.getenv().put(DOT_TESTING_STRING_WITH_COMMA, "VALUE1,VALUE2");
+        System.getenv().put(DOT_TESTING_STRING_WITH_SPACES, "VALUE1 VALUE2");
+        System.getenv().put(DOT_TESTING_STRING, "VALUE_ABC");
+        System.getenv().put(UNABLE_TO_READ_VAR, "NOPE");
+    }
+    
+    
+    
+    @Test
+    public void testing_string_with_comma() {
+
+        String value = Config.getStringProperty("TESTING_STRING_WITH_COMMA");
+        assert(value.equals(Config.getStringProperty("testing.String.with_comma")));
+
+        
+    }
+    
+    
+    
+    @Test
+    public void testing_null_returns() {
+
+        String test = Config.getStringProperty(SIMPLE_TESTING_STRING, null);
+        assertNull(test);
+
+        
+    }
+    
+    
+    @Test
+    public void testing_default_returns() {
+
+        
+        String test = Config.getStringProperty(SIMPLE_TESTING_STRING, TESTING_VALUE);
+        assertEquals(test, TESTING_VALUE);
+
+        
+    }
+    
+    @Test
+    public void testing_notfound_string_returns() {
+
+
+        assert(Config.getStringProperty("no-property") ==null);
+  
+
+
+        
+    }
+
+
+    
+    @Test
+    public void testing_notfound_int_returns() {
+
+        try {
+            Config.getIntProperty("no-property");
+            assert(false);
+        }
+        catch(Exception e) {
+            assert(e instanceof NoSuchElementException);
+            
+        }
+ 
+        
+    }
+
+    @Test
+    public void testing_notfound_float_returns() {
+
+        try {
+            Config.getFloatProperty("no-property");
+            assert(false);
+        }
+        catch(Exception e) {
+            assert(e instanceof NoSuchElementException);
+            
+        }
+ 
+        
+    }
+    
+
+    @Test
+    public void testing_notfound_booean_returns() {
+
+        try {
+            Config.getBooleanProperty("no-property");
+            assert(false);
+        }
+        catch(Exception e) {
+            assert(e instanceof NoSuchElementException);
+            
+        }
+ 
+        
+    }
+    
+    
+    
+    @Test
+    public void test_get_integer_from_env() {
+
+
+        int value =Config.getIntProperty("no-property", -99);
+        assert(value==-99);
+        
+        // this should not work, as we prefix DOT_ to the env variable lookup
+        value =Config.getIntProperty(XXX_TESTING_INTEGER, -99);
+        assert(value==-99);
+        
+        // we should get back Integer.MAX_VALUE, as this get transformed into DOT_TESTING_INTEGER
+        value =Config.getIntProperty("testing.integer", -99);
+        assert(value==Integer.MAX_VALUE);
+    }
+
+    @Test
+    public void test_get_float_from_env() {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5943250)